src/client/styles/date-picker.css

```diff
@@ -1,6 +1,8 @@
 .react-datepicker {
   font-family: unset;
   font-size: 0.9rem;
+  border: none;
+  box-shadow: rgb(0 0 0 / 10%) 0px 4px 6px -1px, rgb(0 0 0 / 6%) 0px 2px 4px -1px;
```

Comment:
used a chakra box-shadow. not sure if there's a better way to interface with Chakra?